src/PubSubHubbub/PubSubHubbub.php

```diff
@@ -108,8 +108,10 @@ public static function clearHttpClient()
      * Set the Escaper instance
      *
      * If null, resets the instance
+     *
+     * @return void
      */
-    public static function setEscaper(Escaper $escaper = null)
+    public static function setEscaper(Escaper $escaper = null): void
```

Comment:
Please do not add php typehints as this will is a BC break unless the class is `final`.

Reply:
Yes, `@return` is fine. Sadly, the native type hint is a problem in case there is a class which is overwriting this method by using inheritance....